packages/cli/src/lib/print-config/print-config-command.unit.test.ts

```diff
@@ -1,23 +1,17 @@
-import { vol } from 'memfs';
-import { beforeEach, describe, expect } from 'vitest';
-import { MEMFS_VOLUME } from '@code-pushup/testing-utils';
+import { describe, expect } from 'vitest';
 import { DEFAULT_CLI_CONFIGURATION } from '../../../mocks/constants';
 import { yargsCli } from '../yargs-cli';
 import { yargsConfigCommandObject } from './print-config-command';
 
-describe('print-config-command', () => {
-  beforeEach(() => {
-    vol.fromJSON(
-      // the real value comes from vitest mocks configured in vitest.config.ts
-
-      {
-        // only needs to exist for stat inside readCodePushupConfig
-        'code-pushup.config.ts': '',
-      },
-      MEMFS_VOLUME,
-    );
-  });
+vi.mock('@code-pushup/core', async () => {
+  const { CORE_CONFIG_MOCK }: typeof import('@code-pushup/testing-utils') =
+    await vi.importActual('@code-pushup/testing-utils');
+  return {
+    readCodePushupConfig: vi.fn().mockResolvedValue(CORE_CONFIG_MOCK),
+  };
```

Comment:
I'd probably preserve the other exported functions from `@code-pushup/core` here as well. Even if they're not called for now, that could change in the future and would lead to pretty strange test error.